Source/DafnyServer/CounterExampleGeneration/DafnyModel.cs

```diff
@@ -458,34 +513,65 @@ public class DafnyModel {
           utfCode = ((Model.Integer)fCharToInt.OptEval(elt)).AsInt();
           return "'" + char.ConvertFromUtf32(utfCode) + "'";
         }
-        return "?#" + GetShortElementId(elt);
+        return GetUnspecifiedValue(elt, "char");
```

Comment:
Sorry, I just noticed this. Does Dafny have some kind of an enum for (primitive) types or something like that?
It is kind of not ideal to be passing strings here. Just curious.

Reply:
Done - I am now using instances of Dafny.Type for this! Is a bit tricky with bit vectors, since they have custom bases, but I believe I made it work.